README.md

```diff
@@ -80,23 +80,23 @@ If you are not familiar with Promise and other ES6 stuff, [get closer to it](htt
 - send money out of Barion via international bank transfer
 - send money to existing Barion account or email address
 
-> **IMPORTANT**: ``node-barion`` is completely consistent with [Barion Docs](https://docs.barion.com/Main_Page), so you can use exactly the same field names, that are specified in it. **It is highly recommended, to read the official Barion documentation** before start to use the ``node-barion`` module.<br>
```

Comment:
Is "It is highly recommended" contains grammar mistake? I developed this package myself, and this is not an official module, so "we" is not makes sense for me.

Reply:
While not strictly grammatically "wrong," it is passive voice which is generally regarded as grammatically inferior. If you would prefer not to use "we" I could switch it back.